### ISSUE TITLE
Nothing happens after the prompt

### ISSUE THREAD

**Issue by jomalla123** [Giftzwerg02/obsidian-quick-xournalpp#2]
Hi, 

I would really like to use your plugin. Unfortunately, while it opens the prompt nothing happens when I click on submit. I also added the xournal bin to my environment variables, but it still does not work.

**Comment by stoneskipping** [comment]
I had the same problem.

Try changing the name of the folder you put the plugin files in to `quick-xournal`, and having a folder in the root of your vault named `resources`.

This is the error that the dev console gave me:
`Error: ENOENT: no such file or directory, copyfile '(vault path)\.obsidian\plugins\quick-xournal\empty.xopp' -> '(vault path)\resources\(my file name).xopp'`

This plugin seems to be mostly incomplete right now, but I hope this helps.

**Comment by jomalla123** [comment]
Thanks, now it works for me. Although, the exported pdf unfortunately does not get updated when I edit the xopp but maybe that features is still missing.

**Comment by Giftzwerg02** [comment]
> This plugin seems to be mostly incomplete right now, but I hope this helps.

Yes it is, this is just a side-project for me which I use(d) for my note-taking in university. Maybe I'll get back to it but I cant say for sure - sorry.

**Comment by stoneskipping** [comment]
> Yes it is, this is just a side-project for me which I use(d) for my note-taking in university. Maybe I'll get back to it but I cant say for sure - sorry.

No worries! Coding takes a lot of time. Thank you for uploading your work, even if you never end up finishing it :-)

**Comment by Giftzwerg02** [comment]
> Thanks, now it works for me. Although, the exported pdf unfortunately does not get updated when I edit the xopp but maybe that features is still missing.

Also I do not think that "syncing" the xopp and pdf file is possible afaik (without further plugins at least). If it is then please tell me since this would definitely be a nice feature!

**Comment by Giftzwerg02** [comment]
> This is the error that the dev console gave me: `Error: ENOENT: no such file or directory, copyfile '(vault path)\.obsidian\plugins\quick-xournal\empty.xopp' -> '(vault path)\resources\(my file name).xopp'`

Are you missing the `empty.xopp` file? If this wasn't downloaded when installing the plugin then you just have to create an empty xournalpp file and move it to the location above. This file will be used as a base file.

Other than that I the plugin *works on my machine* :P (at least from what I've "tested" right now); if there are any more problems just hmu, I am happy to help.

**Comment by jomalla123** [comment]
> > Thanks, now it works for me. Although, the exported pdf unfortunately does not get updated when I edit the xopp but maybe that features is still missing.
> 
> Also I do not think that "syncing" the xopp and pdf file is possible afaik (without further plugins at least). If it is then please tell me since this would definitely be a nice feature!

Not sure, but I guess it is also not too much effort to sync by manually exporting the pdf.

**Comment by Xelbayria** [comment]
I've tried this. It almost works but the exported pdf is not working. I can see it's an unfinished plugin. I've been looking for an alternative to Xournal++ so I can use my pen & tablet to write down and look at it in obsidian. 

Good work. Hope to see the day that plugin will become a finished plugin
